### PR TITLE
Make crosses go first

### PR DESCRIPTION
**Pull request by rastating**
As per the list of issues in the to-do list in the read me, I have changed the code to make the player mark with an X as to follow the rules of tic-tac-toe.

### CODE REVIEW

File: src/TicTacToe.hs [hunk 1]
```diff
@@ -87,25 +87,25 @@ findAndReplace :: Board -> Piece -> Piece -> Board
 findAndReplace b p1 p2 = listToBoard [if x==p1 then p2 else x | x <- bl]
 	where bl = boardToList b
 
--- |Check if X's can immediately win, and if so, do it
+-- |Check if O's can immediately win, and if so, do it
 winningXMove :: Board -> Maybe Board
-winningXMove b = headMay [findAndReplace b p (Right X) | p <- (possibleMoves b), won (findAndReplace b p (Right X))]
+winningXMove b = headMay [findAndReplace b p (Right O) | p <- (possibleMoves b), won (findAndReplace b p (Right O))]
```
Comment (2016rshah) [inline src/TicTacToe.hs]: Thanks for editing the documentation along with the implementation, but the function name also needs to change. I.E `winningXMove` should not be checking if O's can immediately win.

**Review by 2016rshah — COMMENTED** [review]
Thank you for helping out, this is great! If possible, could you also go back and update the function names to accurately represent what they do as well?

File: src/TicTacToe.hs (hunk 1 above)
Comment (rastating) [reply src/TicTacToe.hs]: @2016rshah Sorry, my bad, I thought I had changed them!

Will do it shortly and push the changes up :)

**Comment by 2016rshah** [comment]
Great, thank you so much for your contribution!

**Pull request merged**